packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe core SDK package will be documented in this file.
 
+## Recent Changes
+
+- LTS Breaking: Modified the core SDK. [#2083](https://github.com/zowe/zowe-cli/issues/2083)
```

Comment:
In call changelog references, can we say _why_ we're modifying the core SDK? For performance, for tech currency?

Reply:
The only modification made was to deprecate the IHeaderContent interface to remove duplication between the Core SDK and Imperative, which is captured on the next line. Would it be preferred to remove this line and only have the deprecation line?